Modules/ThirdParty/GDCM/src/gdcm/Source/Common/gdcmSwapper.txx

```diff
@@ -172,7 +176,9 @@ namespace gdcm
     }
   template <> inline float SwapperDoOp::Swap<float>(float val)
     {
-    return static_cast<float>(Swap((uint32_t)val));
+    uint32_t temp;
+    memcpy(&temp, &val, sizeof(uint32_t));
```

Comment:
Failed on Linux
```
2022-01-05T22:29:31.5821948Z /home/vsts/work/1/s/Modules/ThirdParty/GDCM/src/gdcm/Source/Common/gdcmSwapper.txx: In static member function 'static T gdcm::SwapperDoOp::Swap(T) [with T = float]':
2022-01-05T22:29:31.5823350Z /home/vsts/work/1/s/Modules/ThirdParty/GDCM/src/gdcm/Source/Common/gdcmSwapper.txx:180:5: error: 'memcpy' was not declared in this scope
2022-01-05T22:29:31.5824395Z      memcpy(&temp, &val, sizeof(uint32_t));
2022-01-05T22:29:31.5850093Z      ^~~~~~
```

Reply:
Yes, I noticed. I've already submitted a patch upstream.

Reply:
I am afraid that the patch will not work on Linux
```cpp
#if defined(_MSC_VER)

#include <cstring>
```
should be outside `#if defined(_MSC_VER)`

Reply:
Fixed [here](https://github.com/malaterre/GDCM/pull/134)